notebooks/250-efficient-object-detection/250-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>is it possible to provide charts credit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>